src/server/license/server/api_server.go

```diff
@@ -344,13 +346,48 @@ func (a *apiServer) UpdateCluster(ctx context.Context, req *lc.UpdateClusterRequ
 	a.LogReq(req)
 	defer func(start time.Time) { a.pachLogger.Log(req, resp, retErr, time.Since(start)) }(time.Now())
 
-	if err := a.validateClusterConfig(ctx, req.Address); err != nil {
-		return nil, err
+	// should we only validate the pachd Address accessible to the enterprise server?
+	if req.Address != "" {
+		if err := a.validateClusterConfig(ctx, req.Address); err != nil {
+			return nil, err
+		}
+	}
+
+	// is there semantic value to updating a cluster's address or user_address to empty string?
```

Comment:
surfacing this comment/question:

is there semantic value to updating a cluster's address or user_address to empty string?

Reply:
I don't think setting them to an empty string makes sense - in a lot of cases they're probably the same value though? I'm not sure if we can leverage that for updates at all.

Reply:
ok cool 👍  that's what I was thinking too, but wasn't sure if maybe there's something to consider there